etc/ReplayOfflineConfiguration.py

```diff
@@ -132,7 +131,7 @@
 
 # Defaults for GlobalTag
 expressGlobalTag = "130X_dataRun3_Express_v2"
-promptrecoGlobalTag = "130X_dataRun3_Prompt_v2"
+promptrecoGlobalTag = "130X_dataRun3_Prompt_addBSOnline_v1"
```

Comment:
good that it has infinite snapshot :D https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/130X_dataRun3_Prompt_addBSOnline_v1

Reply:
Yes, once we verify that this replay is good, we will make an "offical" v3 GT - which will be an exact copy of this GT.
So I don't think we need yet another replay!

However, if it's the preferred option, I can make v3 already now (I just thought it was better to test it first and make the official GT later).

Reply:
GTs are cheap...

Reply:
done in [0d2bc88](https://github.com/dmwm/T0/pull/4817/commits/0d2bc889d6dd418367ee0f1cbbca0e3fbf0f860f)